pkg/securitycontextconstraints/sccmatching/provider.go

```diff
@@ -101,7 +100,7 @@ func NewSimpleProvider(scc *securityv1.SecurityContextConstraints) (SecurityCont
 func (s *simpleProvider) CreatePodSecurityContext(pod *api.Pod) (*api.PodSecurityContext, map[string]string, error) {
 	sc := securitycontext.NewPodSecurityContextMutator(pod.Spec.SecurityContext)
 
-	annotationsCopy := maps.CopySS(pod.Annotations)
+	annotationsCopy := copySS(pod.Annotations)
```

Comment:
is there no upstream replacement?

Reply:
I didn't find any. The original commit message says that the code was removed because it was unused: https://github.com/kubernetes/kubernetes/commit/488b34eeb2b8f5429358a190f35ae4808b12a995.